lib/site_prism/section.rb

```diff
@@ -52,7 +54,9 @@ def evaluate_script(input)
 
     def parent_page
       candidate_page = parent
-      candidate_page = candidate_page.parent until candidate_page.is_a?(SitePrism::Page)
+      begin
```

Comment:
Why don't we just go back to what was used previously which was `until` at the top. I think it reads nicer.

Reply:
Can we revert this one for now please